From 77a536144551a402166fc7b5ffe08214a9c4c1a9 Mon Sep 17 00:00:00 2001
From: Yongsheng Chen <yongsheng.chen@intel.com>
Date: Mon, 2 Jul 2018 14:34:08 -0700
Subject: [PATCH 110/129] media: intel-ipu4: max9286: Low intensity issue.

Message for Open Source:
Make sure that DIN bit#12 is enabled in CROSSBAR.

Message for Internal:
Make sure that DIN bit#12 is enabled in CROSSBAR.

[Issue/Feature]
Low intensity issue

[Root Cause/Changes]
DIN bit#12 is not enabled in CROSSBAR.

[Test Platform]
 LeafHill

Change-Id: Iea470894a0be703c9cef7a0b471902f2f97b91c5
Tracked-On:#H1407320677
Signed-off-by: Chen, Yongsheng <yongsheng.chen@intel.com>
---
 drivers/media/i2c/max9286-reg-settings.h | 51 ++++++++++++++++----------------
 1 file changed, 25 insertions(+), 26 deletions(-)

diff --git a/drivers/media/i2c/max9286-reg-settings.h b/drivers/media/i2c/max9286-reg-settings.h
index 75dba98..325e375 100644
--- a/drivers/media/i2c/max9286-reg-settings.h
+++ b/drivers/media/i2c/max9286-reg-settings.h
@@ -58,31 +58,30 @@ struct max9286_register_write {
 };
 
 static const struct max9286_register_write max9286_byte_order_settings[] = {
-	{0x20, 0x0C},
-	{0x21, 0x0B},
-	{0x22, 0x0A},
-	{0x23, 0x09},
-	{0x24, 0x08},
-	{0x25, 0x07},
-	{0x26, 0x06},
-	{0x27, 0x05},
-	{0x28, 0x04},
-	{0x29, 0x03},
-	{0x2A, 0x02},
-	{0x2B, 0x01},
-	{0x2C, 0x00},
-	{0x30, 0x1C},
-	{0x31, 0x1B},
-	{0x32, 0x1A},
-	{0x33, 0x19},
-	{0x34, 0x18},
-	{0x35, 0x17},
-	{0x36, 0x16},
-	{0x37, 0x15},
-	{0x38, 0x14},
-	{0x39, 0x13},
-	{0x3A, 0x12},
-	{0x3B, 0x11},
-	{0x3C, 0x10},
+	{0x20, 0x0B},
+	{0x21, 0x0A},
+	{0x22, 0x09},
+	{0x23, 0x08},
+	{0x24, 0x07},
+	{0x25, 0x06},
+	{0x26, 0x05},
+	{0x27, 0x04},
+	{0x28, 0x03},
+	{0x29, 0x02},
+	{0x2A, 0x01},
+	{0x2B, 0x00},
+	{0x30, 0x1B},
+	{0x31, 0x1A},
+	{0x32, 0x19},
+	{0x33, 0x18},
+	{0x34, 0x17},
+	{0x35, 0x16},
+	{0x36, 0x15},
+	{0x37, 0x14},
+	{0x38, 0x13},
+	{0x39, 0x12},
+	{0x3A, 0x11},
+	{0x3B, 0x10},
 };
+
 #endif
-- 
1.9.1

